packages/components/atoms/f-button/test/visual/f-button.visual.desktop.spec.js

```diff
@@ -67,14 +54,9 @@ describe('f-button Desktop visual tests', () => {
     });
 
     describe('link button', () => {
-        beforeEach(() => {
```

Comment:
One thing just to be aware of is we're now using the exact same instance of the Button classes for each test. This might be fine, but it does increase the risk of corrupted/mutated state impacting each subsequent test if any actions are performed on the button.

Again it might not be an issue but could created weirdness down the line.